app/presenters/licences/returns.presenter.js

```diff
@@ -63,6 +66,14 @@ function _returns(returns) {
   })
 }
 
+function _tableCaption(totalAmount, currentAmount) {
```

Comment:
I will add this to a base presenter and look at everywhere we do this and use this new standard presenter.

the last string is unique to the page so can be passed in.